test/python/transpiler/test_transpile_layout.py

```diff
@@ -189,9 +189,6 @@ def test_routing_permutation(self):
             {
                 qr[0]: 2,
                 qr[1]: 4,
-                qr[2]: 1,
-                qr[3]: 0,
-                qr[4]: 3,
```

Comment:
This looks like an unrelated change and it's why CI is failing.

Reply:
Reverted in: https://github.com/Qiskit/qiskit/pull/12221/commits/b93e0cca49213e0aed259d25b9d7376fb6803911

Reply:
Thanks.